setup/native-script.rb

```diff
@@ -42,41 +42,19 @@
 
 puts "Installing the Java SE Development Kit... This might take some time, please, be patient. (You might need to provide your password.)"
 system('brew cask install java')
-system('echo "export JAVA_HOME=$(/usr/libexec/java_home)" >> ~/.bash_profile')
-system('echo "export ANDROID_HOME=/usr/local/opt/android-sdk" >> ~/.bash_profile')
+system('echo "export JAVA_HOME=$(/usr/libexec/java_home)" >> ~/.profile')
 
-puts "Installing node.js 0.12"
-system('brew install homebrew/versions/node012')
+puts "Installing Android SDK"
+system('brew install android-sdk')
+system('echo "export ANDROID_HOME=/usr/local/opt/android-sdk" >> ~/.profile')
```

Comment:
I'm not sure that zsh loads `.profile`: http://superuser.com/questions/187639/zsh-not-hitting-profile